hydra_base/static/default_units_and_dimensions.json

```diff
@@ -1617,6 +1617,7 @@
       },
       {
         "name": "dimensionless",
+        "is_default": "Y",
```

Comment:
didn't we agree to remove the is_default column?

Reply:
forgot it. Removed now